### ISSUE TITLE
Mod doesn't work on RTX A4500

### ISSUE THREAD

**Issue by redhyper9** [Nukem9/dlssg-to-fsr3#187]
Hello, is it possible for the mod to work on an RTX A4500 GPU?

I've installed the mod but DLSS Frame Generation is still grayed out in Cyberpunk 2077.

Here is the mod log file:
```
dlssg-to-fsr3 is loaded. AMD FSR 3 Frame Generation will replace Nvidia DLSS-G Frame Generation.

Please note this is experimental software and is subject to bugs and/or crashes. This does NOT represent a native implementation of AMD's FSR 3.

Report issues at https://github.com/Nukem9/dlssg-to-fsr3

WARNING: DO NOT USE IN MULTIPLAYER GAMES.
NVSDK_NGX_D3D12_Init_Ext
```

Thanks.

**Comment by Styphoryte** [comment]
Yeah, I'm on an RTX 3060 here and have that same exact error log, does your game insta-crash after enabling Frame gen in the in-game settings? Even after disabling and completely removing this mod, still instacrash after trying to open game. Only thing that fixed was manually going to "%LOCALAPPDATA%\CD Projekt Red\Cyberpunk 2077" and then opening UserSettings.json find "DLSS_D" make sure the value is set to false as well as default value and then my game would launch...
